ruby_on_rails-postgres/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+services:
+  db:
+    image: postgres
```

Comment:
We should specifiy the version of the image instead of using the `latest` tag
We can try with the `postgres:12.2-alpine` image